style.css

```diff
@@ -547,3 +547,494 @@ hr {
   border-radius: 10px;
   background-color: black;
 }
+
+.title hr {
+  width: 100%;
+  height: 1px;
+}
+
+/* desktop-version */
+
+@media only screen and (min-width: 760px) {
```

Comment:
- [ ] Please notice that the media icons on your desktop version are not aligned correctly. 
## Your Design
![image](https://user-images.githubusercontent.com/18730106/145455404-a011824f-9495-4901-b278-cb3ebf2e91f2.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/145455432-195fe513-dc07-49d7-aecd-8ccb448c6dde.png)

- [ ] Please also make an adjustment to the background color right now there is a discrepancy between your design background color and the Figma.